src/api/store/event.py

```diff
@@ -477,26 +486,37 @@ def update_event(self, context: Context, args, user_submitted) -> Tuple[dict, Ma
       events.update(**args)
       event: Event = events.first()
 
+      if community_id:
+        community = Community.objects.filter(pk=community_id).first()
+        if community:
+          event.community = community
+        else:
+          event.community = None
```

Comment:
I think this should give an error.  Specified a community_id but ommunity didn't exist